clearwater-memcached/usr/share/clearwater/infrastructure/scripts/memcached

```diff
@@ -25,4 +25,6 @@ sed -e 's/^-l .*$/-l '$listen_address'/g'\
     -e 's/^# *-v *$/-v/g'\
     -e 's/^\(# *\|\)-c.*$/-c 4096/g' </etc/memcached.conf >/etc/memcached_11211.conf
 
+echo "\n# Disable listening on UDP\n-U 0" >> /etc/memcached_11211.conf
+
```

Comment:
Ah, OK you're right about copying the file each time, so ignore my comment about adding the line multiple times (and given this I don't think there is a need to use the utilities).

If you've tested the echo line and confirmed it gives the right output then I guess it's OK, but I'm really not sure how it's working. Could you double check?